net/dnsviz/Portfile

```diff
@@ -19,8 +19,8 @@ long_description    DNSViz is a tool suite for measurement, diagnostic, and \
                     visualization of Domain Name System (DNS) behavior, \
                     including its security extensions (DNSSEC).
 
-checksums           rmd160 ef673e5945605b297b1a5034b59419fde8ad6866 \
-                    sha256 b62e6642aba46cf145f9ca23d02fcfd101752a7448d1b44537334ddc4e359eae
+checksums           rmd160 b64c17ec194824616e542ab704b5df0888bab197 \
+                    sha256 a81ff254c23718cd6f364b03bf6e9c80468fa4663fd5be66043de7b0bece1cab
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
I've (force) pushed a new update that includes `size`.  See if that works better.